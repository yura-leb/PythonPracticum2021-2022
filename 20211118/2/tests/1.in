class C(complex):
    num = Num()

c, d = C(), C()
c.num = 1+2j
d.num = 2 + 4j
print(c.num+d.num)
c.num = (1, 2,3)
print(c.num+d.num)