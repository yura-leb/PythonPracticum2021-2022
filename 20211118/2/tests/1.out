(3+6j)
(5+4j)
